pkgs/applications/video/epgstation/default.nix

```diff
@@ -49,21 +37,14 @@ let
   server = nodePackages.epgstation.override (drv: {
     inherit src;
 
-    bypassCache = false;
-
     # This is set to false to keep devDependencies at build time. Build time
     # dependencies are pruned afterwards.
     production = false;
 
     buildInputs = [ bash ];
-    nativeBuildInputs = [
-      nodejs
-      workaround-opencollective-buildfailures
```

Comment:
The deleted lines IIRC was added to work around build failures. They no longer appear to be necessary.